example_config_class.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+class ThingWorker
+  def self.process(message)
+    FastlyNsq.logger.info("IN PROCESS: #{message}")
+  end
+end
+
+FastlyNsq.configure do |config|
+  config.channel = 'william'
+  # config.logger = Logger.new
+  # config.preprocessor = ->(message) { FastlyNsq.logger.info 'PREPROCESSESES' }
+
+  config.listen_to do |topics|
+    topics.add('assign_supplemental_plans', ThingWorker)
+  end
```

Comment:
This is a little confusing.  `FastlyNsq.configure` will yield FastlyNsq. Then `listen_to` yields `FastlyNsq::Listener::Config.new` (or the existing config). Even charing `topics` here to `listener_config` would help. Or even explicitly calling `FastlyNsq::Listener.config do |listener_config|` might be better. Especially since this is configuring the topics for the listener but isn't actually doing the listening so in that sense `listen_to` is not the best name.